index.qmd

```diff
@@ -1,19 +1,61 @@
-# What They Forgot to Teach You About R {-}
+# What They Forgot to Teach You About R
 
-The initial impetus for creating these materials is a two-day hands-on workshop. The target learner:
+::: {.callout-warning}
+## Warning
+
+This is book is a work in progress.
```

Comment:
This wording isn't quite right. At the very least, remove "is".

But perhaps it's better to convey that it might be a perpetual work in progress? As in, it's living document that tracks what we cover in the companion in-person workshops.

(Although I would not be opposed to WTF becoming something more formal, under the right set of circumstances.)